deploy/manifests/kubernetes.yaml

```diff
@@ -731,6 +838,8 @@ spec:
               value: $(DB_DRIVER)://$(DB_USER):$(DB_PASSWORD)@database:5432/$(DB_NAME)?sslmode=disable
             - name: SERVER_CASDOOR_SERVER
               value: http://identity-access-manager:8000
+            - name: SERVER_S3_SOURCE_ADDRESS
+              value: s3://$(MINIO_ROOT_USER):$(MINIO_ROOT_PASSWORD)@minio:9000/$(MINIO_BUCKET)?sslmode=disable
```

Comment:
Do we need to specify this argument in docker-compose.yaml too?

Reply:
updated